pkg/core/serving/common/common.go

```diff
@@ -77,7 +97,7 @@ func GenOpenFunctionContext(
 	fc := functionContext{
 		Name:    functionName,
 		Version: version,
-		Runtime: strings.Title(strings.ToLower(string(s.Spec.Runtime))),
+		Runtime: cases.Title(language.Und, cases.NoLower).String(string(s.Spec.Runtime)),
```

Comment:
Just curious why we change here? what is the edge case make `strings.Title(...)` not work?

Reply:
It was deprecated in go1.18